core/src/main/resources/com/squarespace/template/plugins/platform/variants-select.html

```diff
@@ -6,7 +6,7 @@
 
   <div class="product-variants" data-item-id="{item.id}" data-variants="{@variants|json|htmltag}"{.if @isSubscribable} data-is-subscribable="true" data-subscription-plan="{@subscriptionPlan|json|htmltag}"{.end} data-animation-role="content">{.repeated section options}
     <div class="variant-option">
-    <div class="variant-option-title">{name|htmltag}: </div>
+    <div class="variant-option-title">{displayName|message name:name|htmltag}: </div>
```

Comment:
Isn't `name` duplicated here ?

Reply:
It is substituting in the name for the field rather than the displayName